pom.xml

```
@@ -15,7 +15,7 @@
 
   <artifactId>TestComplete</artifactId>
   <name>TestComplete support plug-in</name>
-  <version>2.9-SNAPSHOT</version>
+  <version>2.8.2</version>

```

You don't want to change this value, I think.  It needs to retain the -SNAPSHOT suffix.
```suggestion
  <version>2.9-SNAPSHOT</version>

```